tests/api-watch-diff/test.ml

```diff
@@ -126,19 +161,18 @@ let%expect_test "Removing a type" =
     Api_watch_diff.diff_interface ~reference:ref_signature
       ~current:remove_type_signature
   in
-  Format.printf "%b" result;
-  [%expect {|true|}]
+  Format.printf "%a" pp_diff_list result;
+  [%expect {|[Any]|}]
 
 (* Signature for modify_type.mli:
-   > type t = float
-   > type unused_type = string
+   > type t = int
+   > type unused_type = bool
```

Comment:
We can probably rid ourselves of this comment thanks to the test helper function.